informatics-isi-edu/chaise
# Add new record from Record to RecordEdit

This PR addresses #667 and #669. These 2 issues combine to describe a workflow in which:

- A user goes to a Record page and clicks an Add Record link on one of the related tables.
- The link takes the user to the RecordEdit page, where the foreign key relationship between the the main entity on the Record page and the clicked related table is pre-filled.

To test:
1. Go to [this record](https://dev.isrd.isi.edu/~jessie/chaise/record/#1/legacy:dataset/id=4894) on Record app.
2. Click the `Add Record` link for the External Reference related table (at the bottom)
3.  The browser open the RecordEdit app in a new tab, and the url in the new tab should have a `?prefill=` query parameter in the url.
4.  The `id` column on the RecordEdit tab should have the rowname of the main entity from Record page as its pre-filled value.
